docs/notebook-demos/reading-in-sentinel-data.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
biggest picture question to answer at the start of this: why would I want to get this into geopyspark? Seems like I can do a good amount of stuff with rasterio and numpy